### PR TITLE
Console improvements

### PR DESCRIPTION
**Pull request by permcody**
This supplants #5958. 
- Improves the Console indentation regex so that we don't need all the hackery of stripping and re-adding the newline
- Cleans up unnecessary buffer flushes in the EigenExecutioners
- Removes the last piece of duplicated logic in Multiapp.C from a8a56e49 (Why are you always fillingPositions @friedmud :wink:)

### CODE REVIEW

**Comment by permcody** [comment]
I also removed the Magenta. I'm not sure I want that in every Multiapp simulation. I'm not sure I like the Cyan though either. Perhaps we could make it an option if @YaqiWang really wants the color to help him see the transfers better.

**Comment by YaqiWang** [comment]
@permcody I saw one empty line with yak

```
part_solve10: ************************************************************************************
part_solve10: 
part_solve10:                                      YakTest       
part_solve10: 
part_solve10: ************************************************************************************
part_solve10: 
part_solve10: 
part_solve10: 
part_solve10: YAK version:             git commit 5d479d4 on 2015-12-04
part_solve10: 

```

**Comment by friedmud** [comment]
@permcody those damn positions MUST be filled ;-)

**Comment by YaqiWang** [comment]
ic24340
how about maroon, color of TAMU, hehe?

**Comment by jwpeterson** [comment]
> how about maroon, color of TAMU, hehe?

:astonished:

**Comment by jwpeterson** [comment]
@permcody this does not seem to be "under development" any more?

**Comment by permcody** [comment]
Yes, I haven't gotten back to it. While it works for all the MOOSE tests, there's still one extra line when Yaqi prints his headers that I need to look at.

**Comment by permcody** [comment]
This PR should be ready for review once more. I added one additional commit to demonstrate the fix for the header problem @YaqiWang illustrated.

**Comment by YaqiWang** [comment]
let me take a look really quick.

**Comment by YaqiWang** [comment]
Looks good now! We can decide the coloring scheme later.

File: framework/include/outputs/OutputWarehouse.h [hunk 1]
```diff
@@ -33,7 +33,7 @@ class OutputWarehouse : public Warehouse<Output>
   /**
    * Class constructor
    */
-  OutputWarehouse();
+  OutputWarehouse(MooseApp & app);
```
Comment (jwpeterson) [inline framework/include/outputs/OutputWarehouse.h]: There's a MOOSE in the outhouse.

**Pull request merged**